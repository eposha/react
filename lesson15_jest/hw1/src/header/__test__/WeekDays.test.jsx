import React from "react";
import { shallow } from "enzyme";
import WeekDays from "../WeekDays";

describe("<WeekDays/>", () => {
  it("Should display WeekDays", () => {
    const wrappedComponent = shallow(<WeekDays />);
    expect(wrappedComponent).toMatchSnapshot();
  });

  it("Should display WeekDays", () => {
    const wrappedComponent = shallow(<WeekDays />);
    expect(wrappedComponent).toMatchSnapshot();
  });
});
