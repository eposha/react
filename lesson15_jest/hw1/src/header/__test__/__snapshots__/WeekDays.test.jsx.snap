// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WeekDays/> Should display WeekDays 1`] = `
<ul
  className="week"
>
  <li
    className="week__day"
    key="0"
  >
    <span
      className="day-name "
    >
      Mon
    </span>
    <span
      className="day-date"
    >
      17
    </span>
  </li>
  <li
    className="week__day"
    key="1"
  >
    <span
      className="day-name "
    >
      Mon
    </span>
    <span
      className="day-date"
    >
      17
    </span>
  </li>
  <li
    className="week__day"
    key="2"
  >
    <span
      className="day-name "
    >
      Mon
    </span>
    <span
      className="day-date"
    >
      17
    </span>
  </li>
  <li
    className="week__day"
    key="3"
  >
    <span
      className="day-name "
    >
      Mon
    </span>
    <span
      className="day-date"
    >
      17
    </span>
  </li>
  <li
    className="week__day"
    key="4"
  >
    <span
      className="day-name "
    >
      Mon
    </span>
    <span
      className="day-date"
    >
      17
    </span>
  </li>
  <li
    className="week__day"
    key="5"
  >
    <span
      className="day-name "
    >
      Mon
    </span>
    <span
      className="day-date"
    >
      17
    </span>
  </li>
  <li
    className="week__day"
    key="6"
  >
    <span
      className="day-name "
    >
      Mon
    </span>
    <span
      className="day-date"
    >
      17
    </span>
  </li>
</ul>
`;

exports[`<WeekDays/> Should display WeekDays 2`] = `
<ul
  className="week"
>
  <li
    className="week__day"
    key="0"
  >
    <span
      className="day-name "
    >
      Mon
    </span>
    <span
      className="day-date"
    >
      17
    </span>
  </li>
  <li
    className="week__day"
    key="1"
  >
    <span
      className="day-name "
    >
      Mon
    </span>
    <span
      className="day-date"
    >
      17
    </span>
  </li>
  <li
    className="week__day"
    key="2"
  >
    <span
      className="day-name "
    >
      Mon
    </span>
    <span
      className="day-date"
    >
      17
    </span>
  </li>
  <li
    className="week__day"
    key="3"
  >
    <span
      className="day-name "
    >
      Mon
    </span>
    <span
      className="day-date"
    >
      17
    </span>
  </li>
  <li
    className="week__day"
    key="4"
  >
    <span
      className="day-name "
    >
      Mon
    </span>
    <span
      className="day-date"
    >
      17
    </span>
  </li>
  <li
    className="week__day"
    key="5"
  >
    <span
      className="day-name "
    >
      Mon
    </span>
    <span
      className="day-date"
    >
      17
    </span>
  </li>
  <li
    className="week__day"
    key="6"
  >
    <span
      className="day-name "
    >
      Mon
    </span>
    <span
      className="day-date"
    >
      17
    </span>
  </li>
</ul>
`;
